packages/components/src/navigable-container/container.js

```diff
@@ -131,20 +126,19 @@ class NavigableContainer extends Component {
 	}
 
 	render() {
-		const { children, ...props } = this.props;
+		const {
+			children,
+			stopNavigationEvents,
+			eventToOffset,
+			onNavigate,
+			onKeyDown,
+			cycle,
+			onlyBrowserTabstops,
+			forwardedRef,
+			...props
```

Comment:
Nit: perhaps `otherProps` would be clearer?

Reply:
In 3fe1518f700257c16308370aa1c69e8eb5c6aa1d I've used `restProps` which we use here and there for this purpose. Let me know if it works for you.

Reply:
Yup, any indication that it's not the full set of props is fine! 👍